modules/exchanging-identity-token.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-user-access/configure-short-lived-access.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="exchanging-identity-token_{context}"]
+= Exchanging an identity token
+
+[role="_abstract"]
+
+.Prerequisites
+. You have to have a valid OpenID Connect (OIDC) token.
+. A *Machine access* configuration exists for the {product-title-short} instance you want to access.
+
+.Procedure
+. Prepare the POST request's JSON data as in the following example:
++
+[source,json]
+----
+{
+    "idToken": "<id_token>"
+}
+----
+
+. Send a POST request to the API */v1/auth/m2m/exchange*.
+. The response will look like the following example:
++
```

Comment:
See [ISG guidance on writing steps](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#writing-steps) for more info, but basically each step should have a clear command in the first sentence. Here are a few options to do that:

1. Don't write this as a step at all and format it as a result for step 2:
```suggestion
+
.API response
```
2. Write the step as an instruction to wait for a response, here's one way:
```suggestion
. Wait for a response:
+
```